### PR TITLE
Polish: to get rid of "magic" time constants

### PR DESCRIPTION
**Pull request by igor-suhorukov**

### CODE REVIEW

**Comment by jhoeller** [comment]
I like the idea of using `TimeUnit` that way...

File: spring-context-support/src/main/java/org/springframework/scheduling/quartz/SchedulerFactoryBean.java [hunk 1]
```diff
@@ -697,7 +698,7 @@ protected void startScheduler(final Scheduler scheduler, final int startupDelay)
 				@Override
 				public void run() {
 					try {
-						Thread.sleep(startupDelay * 1000);
+						Thread.sleep(TimeUnit.SECONDS.toMillis(startupDelay));
```
Comment (rajadilipkolli) [inline spring-context-support/src/main/java/org/springframework/scheduling/quartz/SchedulerFactoryBean.java]: I would prefer to use TimeUnit.MILLISECONDS.sleep(startupDelay); If you are ok with this suggestion I will submit PR

Comment (igor-suhorukov) [reply spring-context-support/src/main/java/org/springframework/scheduling/quartz/SchedulerFactoryBean.java]: @rajadilipkolli startupDelay in seconds, sorry!

Comment (rajadilipkolli) [reply spring-context-support/src/main/java/org/springframework/scheduling/quartz/SchedulerFactoryBean.java]: @igor-suhorukov Updated comment and this should be fine now.

**Pull request merged**